code/modules/mob/living/carbon/human/species/species.dm

```diff
@@ -123,6 +123,7 @@
 	var/secondary_langs = list()             // The names of secondary languages that are available to this species.
 	var/list/speech_sounds                   // A list of sounds to potentially play when speaking.
 	var/list/speech_chance                   // The likelihood of a speech sound playing.
+	var/list/speech_spans					 // A list of player-optional spans other than normal .say to use when speaking.
```

Comment:
Mostly, yes. I also wanted to make it tracked by-character to make it easier for stuff like clown spans to override it.